microservices-connector/README.md

```diff
@@ -36,83 +36,64 @@ There are `2` components in this repo:
 - 1. `manager`: controller manager to handle GMC CRD
 - 2. `router`: route the traffic among the microservices defined in GMC
 
-#### How to build these binaries?
+### GMC build
+
+#### Binaries building
 
 ```sh
 make build
 ```
 
-#### How to build docker images for these 2 components?
+#### Docker images building
 
 ```sh
 make docker.build
 ```
 
-#### How to delete these components' binaries?
+#### Binaries deleting
 
 ```sh
 make clean
 ```
 
-### To Deploy on the cluster
+### GMC Deployment on K8s cluster
 
-**Build and push your image to the location specified by `CTR_IMG`:**
+**GMC Images NOTE:** This image ought to be published in [dockhub](https://hub.docker.com/u/opea), including [gmcmanager](https://hub.docker.com/r/opea/gmcmanager) and [gmcrouter](https://hub.docker.com/r/opea/gmcrouter). Make sure you have the proper permission to the registry and use the latest images.
```

Comment:
change `[dockhub]` to `[OPEA docker hub]`

Reply:
thanks, done!